_games/2025/2025-04-27-kosei.html

```diff
@@ -0,0 +1,33 @@
+---
+date: 2025-04-27
+season: spring
+game_name: 令和7年度 春季都大会 WEEK3
+place: 佼成学園グラウンド
```

Comment:
過去のものとの統一性をもたせるため、　`佼成学園総合グラウンド` が良さそうです。

```suggestion
place: 佼成学園総合グラウンド
```

Reply:
指摘通り修正しました